python/tests/core/ecl/test_geertsma.py

```diff
@@ -61,3 +72,37 @@ def test_geertsma_kernel():
 
             dz = subsidence.evalGeertsma("S1", None, (1000, 1000, 1900), youngs_modulus, poisson_ratio)
             np.testing.assert_almost_equal(dz, 6.530473913611929e-05, 15)
+
+    @staticmethod
+    def test_geertsma_kernel_2_source_points_2_vintages():
+        grid = EclGrid.createRectangular(dims=(2, 1, 1), dV=(100, 100, 100))
+
+        with TestAreaContext("Subsidence"):
+            p1 = [1, 10]
+            p2 = [10, 20]
+            create_restart(grid, "TEST", p1, p2)
+            create_init(grid, "TEST")
+
+            init = EclFile("TEST.INIT")
+            restart_file = EclFile("TEST.UNRST")
+
+            restart_view1 = restart_file.restartView(sim_time=datetime.date(2000, 1, 1))
+            restart_view2 = restart_file.restartView(sim_time=datetime.date(2010, 1, 1))
+
+            subsidence = EclSubsidence(grid, init)
+            subsidence.add_survey_PRESSURE("S1", restart_view1)
+            subsidence.add_survey_PRESSURE("S2", restart_view2)
+
+            youngs_modulus = 5E8
+            poisson_ratio = 0.3
+
+            dz1 = subsidence.evalGeertsma("S1", None, (1000, 1000, 0), youngs_modulus, poisson_ratio)
+            np.testing.assert_almost_equal(dz1, -5.538064265738908e-05, 15)
```

Comment:
Reduce the precision to e.g. 15 decimals; with this extreme precision we might get false positives due to e.g. platform differences.